data/__init__.py

```diff
@@ -151,377 +151,403 @@
+                         "Cho": [u"Cho", u"Cho /// Cho", u"Cho /// Cho", u"Cho", u"Cho /// Cho", u"Cho /// Cho",
+                                 u"Cho", u"\ucd08 /// Cho", u"Czo", u"\u0427\u043e /// Cho", u"\u53e4 /// Cho",
+                                 u"\u4e18"],
+                         "Chromie": [u"Chromie", u"Cromi /// Chromie", u"Crona /// Chromie", u"Chromie",
+                                     u"Cromi /// Chromie", u"Chromie", u"Cromie", u"\ud06c\ub85c\ubbf8 /// Chromie",
+                                     u"Chronia", u"\u0425\u0440\u043e\u043c\u0438 /// Chromie",
+                                     u"\u514b\u7f57\u7c73 /// Chromie", u"\u514b\u7f85\u7c73 /// Chromie"],
+                         "Johanna": [u"Crusader", u"Johanna /// Johanna", u"Johanna /// Johanna", u"Johanna",
+                                     u"Johanna /// Johanna", u"Johanna /// Johanna", u"Johanna /// Johanna",
+                                     u"\uc694\ud55c\ub098 /// Johanna", u"Johanna /// Johanna",
+                                     u"\u0414\u0436\u043e\u0430\u043d\u043d\u0430 /// Johanna",
+                                     u"\u4e54\u6c49\u5a1c /// Johanna", u"\u55ac\u5b89\u5a1c /// Johanna"],
+                         "D.Va": [u"DVa", u"D.Va /// D.Va", u"D.Va", u"D.Va /// D.Va", u"D.Va", u"D.Va",
+                                  u"D.VA", u"D.Va /// D.Va", u"D.Va", u"D.Va /// D.Va", u"D.Va",
+                                  u"D.Va /// D.Va"],
+                         "Dehaka": [u"Dehaka", u"Dehaka /// Dehaka", u"Dehaka", u"Dehaka", u"Dehaka /// Dehaka",
+                                    u"Dehaka", u"Dehaka", u"\ub370\ud558\uce74 /// Dehaka", u"Dehaka",
+                                    u"\u0414\u0435\u0445\u0430\u043a\u0430 /// Dehaka",
+                                    u"\u5fb7\u54c8\u5361 /// Dehaka", u"\u5fb7\u54c8\u5361 /// Dehaka"],
+                         "Valla": [u"DemonHunter", u"Valla", u"Valla /// Valla", u"Valla /// Valla",
+                                   u"Vala /// Valla", u"Valla /// Valla", u"Valla", u"\ubc1c\ub77c /// Valla",
+                                   u"Valla /// Valla", u"\u0412\u0430\u043b\u043b\u0430",
+                                   u"\u7ef4\u62c9 /// Valla", u"\u7dad\u62c9 /// Valla"],
+                         "Diablo": [u"Diablo", u"Diablo /// Diablo", u"Diablo /// Diablo", u"Diablo /// Diablo",
+                                    u"Diablo /// Diablo", u"Diablo /// Diablo", u"Diablo /// Diablo",
+                                    u"\ub514\uc544\ube14\ub85c /// Diablo", u"Diablo /// Diablo",
+                                    u"\u0414\u0438\u0430\u0431\u043b\u043e /// Diablo",
+                                    u"\u8fea\u4e9a\u6ce2\u7f57 /// Diablo", u"\u8fea\u4e9e\u5e03\u7f85 /// Diablo"],
+                         "Lunara": [u"Dryad", u"Lunara /// Lunara", u"Lunara /// Lunara", u"Lunara /// Lunara",
```

Comment:
I don't get why you replaced the key from Dryad to Lunara.

Reply:
The keys are now the "canonical" name of the hero - so Lunara should be placed there. This is the same logic as if previous - if Dryad was the key and "Dryad" as inputted, we still would have outputted Lunara - see the previous code here: https://github.com/crorella/hots-parser/pull/5/files/ee572c1086ccffd6a211b41d2e3579db1608f621#diff-740b116474b5b8b4d5927ca113397b13L18